docs/DevGuide/BuildSystem.md

```diff
@@ -206,11 +208,34 @@ cmake -D FLAG1=OPT1 ... -D FLAGN=OPTN <SPECTRE_ROOT>
   - This is useful for drastically reducing the build size in CI, but since the
     object files are replaced with empty stubs will generally cause linking
     problems if used during development.
-- SPECTRE_UNIT_TEST_TIMEOUT_FACTOR, SPECTRE_INPUT_FILE_TEST_TIMEOUT_FACTOR and
-  SPECTRE_PYTHON_TEST_TIMEOUT_FACTOR
-  - Multiply the timeout for the respective set of tests by this factor (default
-    is `1`).
-  - This is useful to run tests on slower machines.
+- UBSAN_INTEGER
+  - Whether or not to turn on the undefined behavior sanitizer
```

Comment:
Maybe link to the sanitizer so you can learn how the int, undefined sanitizers differ? Also maybe summarize how this differs from the other undefined behavior sanitizer?

Reply:
I'm adding the links. I'm not sure of a good way to summarize it. I think for something this technical it's best for people to read Clang's documentation, which is fairly short.